routes/chat.js

```diff
@@ -133,6 +140,9 @@ router.post('/chat/:chatId/messages', async function (req, res) {
     delivered_at: new Date(),
   });
 
+  // socket io emit to all users in the chat room that a new message has been created and send the new message
+  io.to(req.params.chatId).emit('messages:create', newMessage);
```

Comment:
create a function with such functionality in `serverSocket.js` and export it.
Trying to keep the socket functionality in `serverSocket.js` and use it elsewhere with encapsulating functions